galsim/phase_screens.py

```diff
@@ -0,0 +1,623 @@
+import numpy as np
```

Comment:
I'll take this comment, and the fact that Mike added the license text, as implicit approval for the way I split up phase_psf.py.  :)